### PR TITLE
feat(content) Quarg: Giving their engines unique flavor & capabilities

### PR DESCRIPTION
**Pull request by Zitchas**
**Content (Artwork / Missions / Jobs)**

## Credit
Thanks to  @Hurleveur for the initial concept and work on this in #7544 , and for giving me permission to utilize their work in creating this. This is not a direct copy, but is based on their work and incorporates much of it. 

## Summary
Quarg stuff in general is bland and boring, and frankly uninspiring for what they are. While all their outfits and ships are planned to be updated as a whole once all the rest of the factions have been added to the game, I think it is fitting that we should be at least establishing the general feel and interest of their stuff.

As such, this PR takes the Quarg engines - which were good, but boring, and given them an actual unique effect that is both fitting to their reputed technological level, and quite simply really interesting juxtaposition of effects. It also buffs the thrusting and turning heat, truly making good on the "extreme efficiency" mentioned in the description.

## Save File
This save file can be used to play through the new mission content:
{{attach a save file that allows people to easily test your added mission content or see your new in-game art}}

## Artwork Checklist
 - none

### CODE REVIEW

**Review by Hurleveur — APPROVED** [review]
I like what you did here, they are now way cooler and give some proper defence against hit force weapons meaning that the engines are made more difficult and rewarding to obtain at the same time! Also its not as trivial as before to run away from them, and it makes these engines useful for moving big ships around.
Also it makes sense to show they actually have very good heat efficiency, at the cost of power, as the thing with Quarg stuff is supposed to be they run quite cold.

Ofc I'm no balance/lore/content expert or reviewer but I still like this change a lot. Inertia reduction was coded with Quarg in mind.

File: data/quarg/quarg outfits.txt [hunk 1]
```diff
@@ -115,9 +115,11 @@ outfit "Medium Graviton Thruster"
 	"engine capacity" -70
 	"thrust" 80
 	"thrusting energy" 20
-	"thrusting heat" 10
+	"thrusting heat" 2
+	"force protection" 0.15
+	"inertia reduction" 0.15
 	"flare sprite" "effect/medium graviton flare"
-	description "Quarg thrusters are extremely powerful for their size, but at the cost of consuming immense amounts of energy."
+	description "Quarg thrusters are quite powerful for their size and produce very little heat, but at the cost of consuming immense amounts of energy. Quarg engines are somehow capable of making their ship act as if it was heavier when affected by external hit forces, but also lighter when it is moving the ship"
```
Comment (Amazinite) [inline data/quarg/quarg outfits.txt:122]:
```suggestion
	description "Quarg thrusters are quite powerful for their size and produce very little heat, but at the cost of consuming immense amounts of energy."
	description "	They also have strange capabilities which seem to violate the laws of physics as humanity understands them. Ships with these engines accelerate faster under their own power than one would expect given the thrust of their engines, while also taking less impact from external forces."
```

File: data/quarg/quarg outfits.txt [hunk 2]
```diff
@@ -128,9 +130,11 @@ outfit "Medium Graviton Steering"
 	"engine capacity" -50
 	"turn" 1600
 	"turning energy" 10
-	"turning heat" 5
+	"turning heat" 1
+	"force protection" 0.15
+	"inertia reduction" 0.15
 	"steering flare sprite" "effect/medium graviton flare"
-	description "Quarg steering engines are as similarly impressive as their thrusters in how much energy they consume. The Quarg don't seem to be bothered by this cost though, given the unmatched strength of their reactors."
+	description "Quarg steering engines are as similarly impressive as their thrusters in how much energy they consume. The Quarg don't seem to be bothered by this cost though, given the unmatched strength of their reactors. Like their thrusters, Quarg steering engines are also capable of making their ship respond to external impacts as if it was much more massive, while simultaneously responding to internal propulsive forces as if it were much lighter."
```
Comment (Amazinite) [inline data/quarg/quarg outfits.txt:137]:
```suggestion
	description "Quarg steering engines are as similarly impressive as their thrusters in how much energy they consume. The Quarg don't seem to be bothered by this cost though, given the unmatched strength of their reactors."
	description "	They also have strange capabilities which seem to violate the laws of physics as humanity understands them. Ships with these engines accelerate faster under their own power than one would expect given the thrust of their engines, while also taking less impact from external forces."
```

**Review by Amazinite — COMMENTED** [review]
The descriptions feel lacking.

**Pull request merged**